Common/Conf.sh

```diff
@@ -62,15 +62,10 @@ TEST_CASES_MAP["501"]="bl51_boxpc"
 TEST_CASES_MAP["502"]="bl70_boxpc"
 TEST_CASES_MAP["700"]="dc19_panelpc"
 
-# Address of Target that will be tested
-MenPcIpAddr="11.10.10.12"
-
 # Credentials for Pc that will be tested - required by ssh connection and sudo cmds
 MenPcLogin="men"
```

Comment:
As we don't use the user:password duple for ssh, could be interesting to remove this line